docs/root/intro/arch_overview/advanced/attributes.rst

```diff
@@ -0,0 +1,153 @@
+.. _arch_overview_attributes:
+
+Attributes
+==========
+
+Attributes refer to contextual properties provided by Envoy during request and
+connection processing. They are named by a dot-separated path (e.g.
+`request.path`), have a fixed type (e.g. `string` or `int64`), and may be
+absent or present depending on the context. Attributes are exposed to CEL
+runtime in :ref:`RBAC filter <arch_overview_rbac>`, as well as Wasm extensions
+via `get_property` ABI method.
+
+Request attributes
+------------------
+
+The following request attributes are generally available upon initial request
+processing, which makes them suitable for RBAC policies:
+
+.. csv-table::
+   :header: Attribute, Type, Description
+   :widths: 1, 1, 4
+
+   request.path, string, The path portion of the URL
+   request.url_path, string, The path portion of the URL without the query string
+   request.host, string, The host portion of the URL
+   request.scheme, string, The scheme portion of the URL e.g. "http"
+   request.method, string, Request method e.g. "GET"
+   request.headers, "map<string, string>", All request headers indexed by the lower-cased header name
+   request.referer, string, Referer request header
+   request.useragent, string, User agent request header
+   request.time, timestamp, Time of the first byte received
```

Comment:
What is the time format?

Reply:
Timestamp and duration are native data types in CEL, with standard functions to extract time of day, etc (see https://github.com/google/cel-spec/blob/master/doc/langdef.md#standard-definitions).

Reply:
I don't think this is super self-explanatory when we're using these same attributes across CEL, Wasm and ext_proc. Would it be possible to provide a standalone data model or a section on how the types translate?

Reply:
Yeah, I can formalize the types for values. I left out Wasm encoding for Wasm documentation.